libraries/AC_Autorotation/AC_Autorotation.cpp

```diff
@@ -307,34 +307,35 @@ bool AC_Autorotation::get_norm_head_speed(float& norm_rpm) const
 }
 
 // Update speed controller
-void AC_Autorotation::update_forward_speed_controller(float pilot_norm_accel)
+void AC_Autorotation::update_forward_speed_controller(float pilot_accel_norm)
 {
     // Limiting the desired velocity based on the max acceleration limit to get an update target
-    const float min_vel = _target_vel - get_accel_max() * _dt;
-    const float max_vel = _target_vel + get_accel_max() * _dt;
-    _target_vel = constrain_float(_desired_vel, min_vel, max_vel); // (m/s)
+    const float min_vel_ms = _target_vel_ms - get_accel_max_mss() * _dt;
+    const float max_vel_ms = _target_vel_ms + get_accel_max_mss() * _dt;
+    _target_vel_ms = constrain_float(_desired_vel_ms, min_vel_ms, max_vel_ms); // (m/s)
 
     // Calculate acceleration target
-    const float fwd_accel_target  = _fwd_speed_pid.update_all(_target_vel, get_speed_forward(), _dt, _limit_accel); // (m/s/s)
+    const float fwd_accel_target_mss  = _fwd_speed_pid.update_all(_target_vel_ms, get_speed_forward_ms(), _dt, _limit_accel); // (m/s/s)
 
     // Build the body frame XY accel vector.
     // Pilot can request as much as 1/2 of the max accel laterally to perform a turn.
     // We only allow up to half as we need to prioritize building/maintaining airspeed.
-    Vector2f bf_accel_target = {fwd_accel_target, pilot_norm_accel * get_accel_max() * 0.5};
+    Vector2f bf_accel_target_mss = {fwd_accel_target_mss, pilot_accel_norm * get_accel_max_mss() * 0.5};
 
     // Ensure we do not exceed the accel limit
-    _limit_accel = bf_accel_target.limit_length(get_accel_max());
+    _limit_accel = bf_accel_target_mss.limit_length(get_accel_max_mss());
 
     // Calculate roll and pitch targets from angles, negative accel for negative pitch (pitch forward)
-    Vector2f angle_target = { accel_to_angle(-bf_accel_target.x), // Pitch
-                              accel_to_angle(bf_accel_target.y)}; // Roll
+    Vector2f angle_target_deg = { accel_mss_to_angle_deg(-bf_accel_target_mss.x), // Pitch
+                              accel_mss_to_angle_deg(bf_accel_target_mss.y)}; // Roll
```

Comment:
minor white space change here, also not a blocker

FYI @MattKear @Ferruccio1984, AutoRotation mode is having some functions, arguments and internal variables renamed in this PR